types/sharepoint/tslint.json

```diff
@@ -18,6 +17,7 @@
         "no-unnecessary-qualifier": false,
         "no-unnecessary-type-assertion": false,
         "prefer-template": false,
-        "use-default-type-parameter": false
+        "use-default-type-parameter": false,
```

Comment:
Instead of globally disabling these two lint rules, could you please disable them on a line-by-line basis?

Reply:
Done